Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.8.2-buster as base
+
+RUN pip install gunicorn
```

Comment:
Gunicorn can be managed by poetry

Reply:
As Alex pointed out, you do have gunicorn listed in your pyproject.toml file. So you can remove this line.